jobspec/parse.go

```diff
@@ -452,6 +462,75 @@ func parseTasks(result *[]*structs.Task, list *ast.ObjectList) error {
 	return nil
 }
 
+func parseServices(jobName string, taskGroupName string, task *structs.Task, serviceObjs *ast.ObjectList) error {
+	task.Services = make([]structs.Service, len(serviceObjs.Items))
+	for idx, o := range serviceObjs.Items {
+		var service structs.Service
+		var m map[string]interface{}
+		if err := hcl.DecodeObject(&m, o.Val); err != nil {
+			return err
+		}
+
+		delete(m, "check")
+
+		if err := mapstructure.WeakDecode(m, &service); err != nil {
+			return err
+		}
+
+		if idx > 0 && service.Name == "" {
```

Comment:
Shouldn't the check be that you have more than one empty name service block. Like I might want one that is has the default name and the rest don't.